tests/test_delineateit.py

```diff
@@ -487,16 +526,21 @@ def test_check_geometries(self):
         target_vector_path = os.path.join(
             self.workspace_dir, 'checked_geometries.gpkg')
         with self.assertRaises(ValueError) as cm:
-            delineateit.check_geometries(
+            delineateit.preprocess_geometries(
                 outflow_vector_path, dem_raster_path, target_vector_path,
                 skip_invalid_geometry=False
             )
         self.assertTrue('is invalid' in str(cm.exception))
 
-        delineateit.check_geometries(
-            outflow_vector_path, dem_raster_path, target_vector_path,
-            skip_invalid_geometry=True
-        )
+        logger = logging.getLogger('natcap.invest.delineateit.delineateit')
+        with capture_logging(logger) as log_records:
+            delineateit.preprocess_geometries(
+                outflow_vector_path, dem_raster_path, target_vector_path,
+                skip_invalid_geometry=True
+            )
+        self.assertEqual(len(log_records), 6)
+        self.assertIn('named "ws_id". Field values will be overwritten',

```

Comment:
This message could be made a module-level variable in `delineateit` and referenced here. But it's not a lot of duplication.

Reply:
Yep, sure can!  Added in a055e59ce